### ISSUE TITLE
Default resource of static content

### ISSUE THREAD

**Issue by wiwiwa** [ktorio/ktor#1855]
**Ktor Version and Engine Used (client or server and name)**
1.3.1

**Describe the bug**
Ktor response 404 when requesting a default `index.html` in sub package, for example '/test/'. It should return the content of default index.html.
If we don't have this feature, we can not package a web application as a single executable jar.

**To Reproduce**
1. Place a `index.html` file in package `web.test`. Of course we have other files in package `web`.
2. configure static resource use the following code:
```kotlin
        routing {
            static{
                resources("web")
                defaultResource("index.html","web")
            }
        }

```
3. Access <http://localhost:8080/test/> returns 404

**Expected behavior**
Access <http://localhost:8080/test/> should return the content of file `index.html` in package `web.test`

Attached a simple test project source files
[ktor-test.zip](https://github.com/ktorio/ktor/files/4598763/ktor-test.zip)

**Comment by oleg-larshin** [comment]
Please check the <a href="https://youtrack.jetbrains.com/issue/KTOR-387">following ticket</a> on YouTrack for follow-ups to this issue. GitHub issues will be closed in the coming weeks.

**Comment by e5l** [comment]
Fixed since 1.5.0
